test/pages/CheckoutStepOne.page.js

```diff
@@ -0,0 +1,23 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+class CheckoutStepOnePage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    get firstName() { return $('#first-name') };
+
+    get lastName() { return $('#last-name') };
+
+    get postalCode() { return $('#postal-code') };
+
+    get continueBtn() { return $('#continue') };
+
+    async fillInCheckout(firstName, lastName, postalCode) {
```

Comment:
I'd rather use an object here:
```suggestion
    async fillInCheckout({firstName, lastName, postalCode}) {
```